<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CMS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="recGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Receive Messages</string>
    </property>
    <widget class="QPushButton" name="bttnRecDel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnRecDetails">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>72</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnRecView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>72</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>View/Play</string>
     </property>
    </widget>
    <widget class="QColumnView" name="recMsgList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>401</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Coded Message System</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matthew Hengeveld</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark Whipple</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="modeGroup">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>MODE</string>
    </property>
    <widget class="QRadioButton" name="radioHome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Home Station</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioRec">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Receiver</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="sendTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="sendTab">
     <attribute name="title">
      <string>Send Messages</string>
     </attribute>
     <widget class="QPushButton" name="bttnSaveText">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Text Message</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textMsg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Message</string>
      </property>
     </widget>
     <widget class="QListWidget" name="sendRecList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>191</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>New Message</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="bttnSendMsg">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>330</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Message</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bttnAudioDel">
      <property name="geometry">
       <rect>
        <x>331</x>
        <y>170</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bttnAudioPlay">
      <property name="geometry">
       <rect>
        <x>331</x>
        <y>120</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bttnAudioRec">
      <property name="geometry">
       <rect>
        <x>331</x>
        <y>70</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="recSign">
      <property name="geometry">
       <rect>
        <x>321</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IDLE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="sendSign">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IDLE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="setTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QSlider" name="recLenSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QComboBox" name="recBitrateBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>8 bps</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 bps</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="recSamRateBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>8000 Hz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16000 Hz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32000 Hz</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sample Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bitrate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Recording Length (seconds):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="recLenLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>33</y>
        <width>47</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>20</y>
      <width>351</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>53</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serialPortDropbox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="baudrateDropbox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="timeoutDropbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>250</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>750</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1250</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1750</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>25</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout (ms)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QCheckBox" name="enEncrypt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Encryption</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="enCompress">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Compression</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCMS">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHome_Station_Mode"/>
    <addaction name="actionReceiver_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMessage">
    <property name="title">
     <string>Message</string>
    </property>
    <addaction name="actionSend_Audio_Message"/>
    <addaction name="separator"/>
    <addaction name="actionPlay_Audio"/>
    <addaction name="actionRecord_Audio"/>
    <addaction name="actionDelete_Audio"/>
    <addaction name="separator"/>
    <addaction name="actionSend_Text_Message"/>
    <addaction name="separator"/>
    <addaction name="actionView_Play_Message"/>
    <addaction name="actionMessage_Details"/>
    <addaction name="actionDelete_Message"/>
   </widget>
   <addaction name="menuCMS"/>
   <addaction name="menuMessage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRecord_Audio">
   <property name="text">
    <string>Record Audio</string>
   </property>
  </action>
  <action name="actionPlay_Audio">
   <property name="text">
    <string>Play Audio</string>
   </property>
  </action>
  <action name="actionSend_Audio_Message">
   <property name="text">
    <string>Send Audio Message</string>
   </property>
  </action>
  <action name="actionSend_Text_Message">
   <property name="text">
    <string>Send Text Message</string>
   </property>
  </action>
  <action name="actionView_Play_Message">
   <property name="text">
    <string>View/Play Message</string>
   </property>
  </action>
  <action name="actionMessage_Details">
   <property name="text">
    <string>Message Details</string>
   </property>
  </action>
  <action name="actionDelete_Message">
   <property name="text">
    <string>Delete Message</string>
   </property>
  </action>
  <action name="actionHome_Station_Mode">
   <property name="text">
    <string>Home Station Mode</string>
   </property>
  </action>
  <action name="actionReceiver_Mode">
   <property name="text">
    <string>Receiver Mode</string>
   </property>
  </action>
  <action name="actionDelete_Audio">
   <property name="text">
    <string>Delete Audio</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
